hail/src/test/scala/is/hail/expr/ir/IRSuite.scala

```diff
@@ -742,6 +740,769 @@ class IRSuite extends HailSuite {
     assertEvalsTo(MakeArray(FastSeq(), TArray(TInt32())), FastIndexedSeq())
   }
 
+  @Test def testMakeArrayInferPTypeFromNestedRef() {
```

Comment:
I don't think we need this anymore

Reply:
I think you need this test, and not the InferMakeArrayNestedIR one.

Though this could also just be merged with the `testMakeArrayInferPType` test.